src/helperFunctions/program_setup.py

```diff
@@ -58,14 +60,15 @@ def _get_console_output_level(debug_flag):
     return logging.INFO
 
 
-def _setup_logging(config, args):
+def _setup_logging(config, args, component):
```

Comment:
A default value for component would make sense and prevent one test from failing
```suggestion
def _setup_logging(config, args, component=None):
```